tests/bin/install-wp-tests.sh

```diff
@@ -121,6 +119,9 @@ install_test_suite() {
 		sed $ioption "s/yourusernamehere/$DB_USER/" "$WP_TESTS_DIR"/wp-tests-config.php
 		sed $ioption "s/yourpasswordhere/$DB_PASS/" "$WP_TESTS_DIR"/wp-tests-config.php
 		sed $ioption "s|localhost|${DB_HOST}|" "$WP_TESTS_DIR"/wp-tests-config.php
+
+		# Enforce mysqli.
+		echo "define( 'WP_USE_EXT_MYSQL', false );" >> "$WP_TESTS_DIR"/wp-tests-config.php
```

Comment:
Just noting, for any future reviewers/on-lookers, that this change removes the link to a `master` repo URL and that the file being downloaded does what this new code does. If anything, this is probably more stable in case that repo is removed 🙂

Reply:
@tofumatt @felixarntz I don't think we should bother making customizations to this file. This script is essentially an [external dependency](https://github.com/wp-cli/scaffold-command/blob/479cd4f10f638d7a46a5b4a60c80bd4b4a6bd5e2/templates/install-wp-tests.sh) that we mirror in our source because it's a shell script.

If anything, this patch should be an upstream contribution 🙂 

In the future, we should be able to update this by replacing this file with the newest version from upstream without requiring manual updates afterwards.